docs/Glossary

```diff
@@ -0,0 +1,42 @@
+Glossary
+========
+
++-----------------------+----------------------------------------------+
```

Comment:
Rather than using a table, the glossary should use a [definition list](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists), which is a syntax that is specially suited for e.g. a glossary.

Reply:
That's done now.